Cargo.toml

```diff
@@ -54,31 +54,31 @@ alloc = [
 
 [dependencies]
 async-attributes = { version = "1.1.1", optional = true }
-async-task = { version = "1.2.1", optional = true }
+async-task = { version = "1.3.1", optional = true }
 broadcaster = { version = "1.0.0", optional = true }
-crossbeam-channel = { version = "0.4.0", optional = true }
-crossbeam-deque = { version = "0.7.2", optional = true }
-crossbeam-utils = { version = "0.7.0", optional = true }
-futures-core = { version = "0.3.1", optional = true, default-features = false }
-futures-io = { version = "0.3.1", optional = true }
+crossbeam-channel = { version = "0.4.2", optional = true }
+crossbeam-deque = { version = "0.7.3", optional = true }
+crossbeam-utils = { version = "0.7.2", optional = true }
+futures-core = { version = "0.3.4", optional = true, default-features = false }
+futures-io = { version = "0.3.4", optional = true }
 futures-timer = { version = "2.0.2", optional = true }
 kv-log-macro = { version = "1.0.4", optional = true }
 log = { version = "0.4.8", features = ["kv_unstable"], optional = true }
-memchr = { version = "2.3.0", optional = true }
+memchr = { version = "2.3.3", optional = true }
 mio = { version = "0.6.19", optional = true }
 mio-uds = { version = "0.6.7", optional = true }
-num_cpus = { version = "1.11.1", optional = true }
-once_cell = { version = "1.2.0", optional = true }
-pin-project-lite = { version = "0.1.2", optional = true }
+num_cpus = { version = "1.12.0", optional = true }
+once_cell = { version = "1.3.1", optional = true }
+pin-project-lite = { version = "0.1.4", optional = true }
 pin-utils = { version = "0.1.0-alpha.4", optional = true }
 slab = { version = "0.4.2", optional = true }
 
 [dev-dependencies]
 femme = "1.3.0"
 rand = "0.7.3"
-surf = "1.0.3"
+surf = "2.0.0-alpha.0"
```

Comment:
I'm not sure it is good to depend on pre-release crates. They sometimes break...